client/src/services/unoService.js

```diff
@@ -4,6 +4,10 @@ export default {
   async getAllGames() {
     return BaseService.get(`/api/games`);
   },
+  
+  async getGameSummary(gameId) {
```

Comment:
Can this not just tie into the existing getGameState endpoint?

Reply:
At this point, the user won't have a JWT since they haven't joined the game yet, and getGameState requires that you have a JWT. It also returns a slightly different data structure; for example, Creator is a Player in getAllGames, but a string in getAllGames or my new function, getGameSummary. Rather than create exceptions to when a JWT is needed and then manipulating one data structure into another, I thought that creating a singular version for the endpoint getAllGames would be simpler.